tests/cmdline/commands/test_rabbitmq.py

```diff
@@ -69,7 +69,7 @@ def test_tasks_revive_without_daemon(run_cli_command):
     assert run_cli_command(cmd_rabbitmq.cmd_tasks_revive, raises=True)
 
 
-@pytest.mark.usefixtures('started_daemon_client')
+@pytest.mark.usefixtures('aiida_profile_clean')
```

Comment:
This can solve the failed test, I think here the problem was caused by the rmq resource racing the queue might be cleaned by the previous test maybe?? 

When I run only this test in CI, the test will pass. Test only fail when it is run with other tests. I guess it is because the daemon_client is shared and the rmq queue is getting cleaned by other tests (high likely the previous one? it is easy to check by just run the test in this file). 

But no matter what the cause is, the solution I think is make the daemon_client independent for this test by using `aiida_profile_clean` fixture which will close the daemon (which will call `broker_reset` that close the rmq before running the test).